src/tests/suite/daos_container.c

```diff
@@ -52,7 +54,28 @@ co_create(void **state)
 				    &info, arg->async ? &ev : NULL);
 		assert_rc_equal(rc, 0);
 		WAIT_ON_ASYNC(arg, ev);
+		assert_int_equal(info.ci_nhandles, 0);
 		print_message("container opened\n");
+
+		/* Open a second time to verify num handles was incremented */
+		rc = daos_cont_open(arg->pool.poh, str, DAOS_COO_RO, &coh2,
+				    &info, arg->async ? &ev : NULL);
+		assert_rc_equal(rc, 0);
+		WAIT_ON_ASYNC(arg, ev);
+		assert_int_equal(info.ci_nhandles, 1);
```

Comment:
same here, since we have 2 handles now, it should be 2?

Reply:
yes, will fix in the next push